modules/custom/az_person/config/install/core.entity_form_display.node.az_person.default.yml

```diff
@@ -3,6 +3,7 @@ status: true
 dependencies:
```

Comment:
Put this file unchanged here: https://github.com/az-digital/az_quickstart/tree/main/modules/custom/az_seo/config/quickstart

Reply:
Then add the same file here without lines:

- https://github.com/az-digital/az_quickstart/pull/1871/files#diff-ec610764f264f1365ff8044c366354a9064ec1b795dd50c10fb1f36672e495e5R16
- https://github.com/az-digital/az_quickstart/pull/1871/files#diff-ec610764f264f1365ff8044c366354a9064ec1b795dd50c10fb1f36672e495e5R29
- https://github.com/az-digital/az_quickstart/pull/1871/files#diff-ec610764f264f1365ff8044c366354a9064ec1b795dd50c10fb1f36672e495e5R163-R170

Reply:
That should resolve the issue.

Reply:
You need the same file twice, once with the metatags, once without.

Reply:
But of course, that means that we need to make az_person a dependency on az_seo (probably)